src/client/index.html

```diff
@@ -24,7 +24,11 @@
 <!-- endinject -->
 </head>
 <body>
-  <div ui-view></div>
+  <div class="navbar">
```

Comment:
OK for now. FYI I'm making a partial view of a navbar so this div will eventually be removed